app/models/group.rb

```diff
@@ -36,20 +36,23 @@ class Group < ActiveRecord::Base
 
   has_one :forum
   has_one :privacy_setting
-  has_many :reqs, :conditions => ["biddable = ?",true], :order => "created_at DESC"
-  has_many :offers, :order => "created_at DESC"
-  has_many :photos, :as => :photoable, :dependent => :destroy, :order => "created_at"
-  has_many :exchanges, -> { order("created_at DESC") }
-  has_many :exchange_and_fees, :order => "created_at DESC"
+  has_many :reqs, -> { where(biddable: true).order(created_at: :desc) }
+  has_many :offers, -> { order(created_at: :desc) }
+  has_many :photos, -> { order(:created_at) }, :as => :photoable, :dependent => :destroy
```

Comment:
I suppose you can use the old hashrocket syntax if you MUST :)

Reply:
Yeah, I didn't want to change too many things at once. Changing hash rocket syntax to JSON syntax can be easily automated with RuboCop.